identifier/identifier_spdx_test.go

```diff
@@ -47,7 +47,7 @@ func Test_identifyLicensesInSPDXTestDataDirectory(t *testing.T) {
 		t.Errorf("IdentifyLicensesInDirectory(%v) err = %v", testDataDir, err)
 	}
 
-	const expected = 541
+	const expected = 553
```

Comment:
Is this because of a new additions in 3.20?

Reply:
yes 12 net new after removing deprecated ones and adding new ones